modules/ROOT/pages/migrate-cl.adoc

```diff
@@ -0,0 +1,49 @@
+* Along with Fedora 31, FCOS does not allow SSH password logins by default. For full details on how to set up users, refer to the documentation for the xref:ign-passwd.adoc[`passwd`] node of the FCC file.
+//* docker version selection
+* Because `usermod` is not yet fully-functional on FCOS, there is a `docker` group in the `/etc/group` file. This is a stop-gap measure to facilitate a smooth transition to FCOS. The team is working on a more functional `usermod`, at which time the `docker` group will no longer be included by default.
+* There is no way to create directories below the `root` directory. Changes are restricted to `/etc` and `/var`. Refer to the documentation for the xref:ign-storage.adoc[`storage`] node of the FCC configuration for details about writing directories and files to FCOS.
+
+== Operator notes
+* CL had three release channels: Stable, Beta, and Alpha. FCOS maintains the Fedora standard of two release streams: test and stable. See the https://getfedora.org/coreos/download/[download page] for more details.
+* Automatic updates are coordinated by Zincati, as described in the https://github.com/coreos/zincati/blob/master/docs/usage/auto-updates.md[Zincati documentation]. Each update requires a system reboot, and https://github.com/coreos/rpm-ostree/blob/master/README.md[`rpm-ostree`] provides a rollback mechanism in the grub menu.
+* The `update-ssh-keys` tool is not shipped on FCOS. The preferred method is to use the https://www.ssh.com/ssh/authorized_keys/openssh/[`AuthorizedKeysFile`].
+//*SELinux policy differences. What are they?
+* In CL, metrics/telemetry data was collected by the update mechanism. In FCOS, the plan is to collect data (without unique identifiers) via https://github.com/coreos/fedora-coreos-pinger/issues/30[`fedora-coreos-pinger`]. Users will have the ability to disable data collection.
+* Cloud CLI clients are not included in FCOS. There is an initiative to create a "tools" container to run on FCOS.
+\\* Sticky directory - https://github.com/coreos/bugs/issues/2577
+* CL left Simultaneous Multi-Threading (SMT) enabled but advised users to turn it off if their systems were vulnerable to certain issues such as https://www.kernel.org/doc/html/latest/admin-guide/hw-vuln/l1tf.html[L1TF] or https://www.kernel.org/doc/html/latest/admin-guide/hw-vuln/mds.html[MDS]. By default, FCOS https://github.com/coreos/fedora-coreos-tracker/issues/181[automatically disables SMT] for vulnerable systems.
+
+== Implementation notes
+//* Partition layout differences. CL is at https://coreos.com/os/docs/latest/sdk-disk-partitions.html. I can't make heads or tails of the results of the discussions in https://github.com/coreos/fedora-coreos-tracker/issues/94.
+* The default filesystem choices on CL were ext4, btrfs, and xfs. On FCOS, the choices are xfs and ext4. If you wish to use btrfs, you can use xref:ign-storage.adoc[Ignition] to reformat `/var` to btrfs.
```

Comment:
The CL default was ext4 and the FCOS default is XFS except for /boot and /boot/efi.